lang/gjs/src/gtk3/astalify.ts

```diff
@@ -35,8 +36,7 @@ export default function astalify<
         get noImplicitDestroy(): boolean { return this[noImplicitDestroy] }
         set noImplicitDestroy(value: boolean) { this[noImplicitDestroy] = value }
 
-        set actionGroup([prefix, group]: ActionGroup) { this.insert_action_group(prefix, group) }
-        set_action_group(actionGroup: ActionGroup) { this.actionGroup = actionGroup }
+        set insertActionGroup([prefix, group]: ActionGroup) { this.insert_action_group(prefix, group) }
```

Comment:
`insertActionGroup` would suggest it's a function, not a property. Keep the `actionGroup` name for the propery.

Reply:
> It does not add the action group to a list, it replaces the action group with the given prefix in the list

Ahh I guess I misunderstood that part. But still, it doesn't make sense to call `actionGroup` a prop when you can have multiple groups with different prefixes and it has no getters. It is still a method that calls `insert_action_group` at then end of the day.

I don't know if it's possible, but maybe doing something like the signal props where it's `on[SomeEvent]` but it would be `actionGroupDbusmenu`? we could then actually have a getter and setter that do act like a normal property

Reply:
> I don't know if it's possible

It was not, in fact, possible.

I'll leave it like it this, makes the PR way easier now.